requirements.txt

```diff
@@ -14,6 +12,9 @@ python-dateutil==2.6.1
 PyYaml==5.1
 six==1.11.0
 ujson==1.35
-yarl==1.2.6
 zope.interface==4.4.3
+# Force build uvloop from source instead of using wheel
+# Replace with a uvicorn version that uses uvloop built for python 3.8
+git+https://github.com/dmontagu/uvloop@patch-1
```

Comment:
^ Temporary fix. I guess this would be fixed in the following days.

Reply:
That may delay the merge